FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Views/EmailAuthView.swift

```diff
@@ -127,7 +127,7 @@ extension EmailAuthView: View {
       .frame(maxWidth: .infinity)
       .buttonStyle(.borderedProminent)
       Button(action: {
-        authService.authView = .passwordRecovery
+        authService.authView = .emailLink
```

Comment:
Should these be two separate buttons?

Reply:
They are two separate buttons because they are two separate Views. 

password recovery: https://github.com/firebase/FirebaseUI-iOS/blob/opt-in-email-provider/FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Views/EmailAuthView.swift#L84

email link sign-in: https://github.com/firebase/FirebaseUI-iOS/blob/opt-in-email-provider/FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Views/EmailAuthView.swift#L129

Which are rendered in the AuthPickerView here: https://github.com/firebase/FirebaseUI-iOS/blob/opt-in-email-provider/FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Views/AuthPickerView.swift#L24-L27

If you prefer a different way then let me know 👍

Reply:
@morganchen12  Also - I have a question in the description regarding whether calling `AuthService()
    .withEmailSignIn()` should also hide email sign-in. I'm 99% sure that it should but just wanted confirmation.

Reply:
Do you mean the opposite, where not calling `.withEmailSignIn()` should hide it? That sounds good to me.

Reply:
sorry, I did mean the opposite. Thanks for clarification 🙏